src/helpers/CameraHelper.js

```diff
@@ -132,14 +132,14 @@ class CameraHelper extends LineSegments {
 
 		const colorAttribute = geometry.getAttribute( 'color' );
 
-		// near
+		// far
```

Comment:
This change does not look correct to me. If you comment out the four lines below, you will see that the near lines won't receive a color.